main.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     // Load initial document
     var defaultHTML = brackets.getModule("text!filesystem/impls/filer/lib/default.html");
+    var defaultCSS  = brackets.getModule("text!filesystem/impls/filer/lib/style.css");
+    var defaultJS   = brackets.getModule("text!filesystem/impls/filer/lib/script.js");
```

Comment:
When I debug this code it just errors out at line 44 when it attempts to get the CSS file